frontend/src/storybook/checkbox/checkbox-story.tsx

```diff
@@ -0,0 +1,19 @@
+import { Checkbox } from '~/bundles/common/components/checkbox/checkbox.js';
+
+import { useAppForm } from '../../bundles/common/hooks/hooks.js';
```

Comment:
use relative path ~/bundles/common/hooks/hooks.js

Reply:
and also import Checkbox from components.js